index.html

```diff
@@ -25,38 +26,32 @@ <h1>Nabla API sample app</h1>
             <div class="filter">
                 <label for="transcript-locale">Transcript locale:</label>
                 <select id="transcript-locale">
-                    <option value="en-US">en-US</option>
-                    <option value="en-GB">en-GB</option>
-                    <option value="fr-FR">fr-FR</option>
-                    <option value="es-ES">es-ES</option>
-                    <option value="es-MX">es-MX</option>
+                    <option value="ENGLISH_US">ENGLISH_US</option>
+                    <option value="ENGLISH_UK">ENGLISH_UK</option>
+                    <option value="FRENCH">FRENCH</option>
```

Comment:
It's `FRENCH_FR`, also for transcript-locale we can now list all the 35 supported locales.
```suggestion
                    <option value="FRENCH_FR">FRENCH_FR</option>
```